post-processor/yandex-export/post-processor.go

```diff
@@ -25,7 +27,12 @@ import (
 type Config struct {
 	common.PackerConfig `mapstructure:",squash"`
 
-	// Paths to Yandex Object Storage where exported image will be uploaded
+	// List of paths to Yandex Object Storage where exported image will be uploaded.
+	// Please be aware that use of space char inside path not supported.
+	// Example value `[
+	//        "s3://bucket-name/image-blob.qcow2",
+	//      ]`
+	// Also this param support [build](/docs/templates/engine) template function.
```

Comment:
Once the build template data section is created I think this link can point to that Yandex specific section with an example.

Reply:
Remove this example and make link to avaliable build variables in Yandex builder.